packages/roosterjs-content-model-core/lib/editor/core/createEditorCore.ts

```diff
@@ -43,7 +43,11 @@ export function createEditorCore(contentDiv: HTMLDivElement, options: EditorOpti
             options.knownColors,
             options.generateColorKey
         ),
-        trustedHTMLHandler: options.trustedHTMLHandler || defaultTrustHtmlHandler,
+        trustedHTMLHandler:
+            options.trustedHTMLHandler && !isDOMCreator(options.trustedHTMLHandler)
+                ? options.trustedHTMLHandler
+                : defaultTrustHtmlHandler,
```

Comment:
If options.trustedHTMLHandler is DOMCreator, we still need to create a valid trustedHTMLHandler, to satisfy the following case:
1. Pass in a DOMCreator when init editor
2. call editor.getTrustedHTMLHandler() and use it to sanitize HTML

You can create a function here to use DOMCreator() to return a DOM tree then get its innerHTML and return

Reply:
Once we integrate this into OWA, we can search all places where getTrustedHTMLHandler is used and replace with getDOMCreator(), so this new function should not really be used in OWA but other roosterjs customers may still need it